src/ImageSharp/Formats/Jpeg/Components/Decoder/IJpegComponent.cs

```diff
@@ -10,6 +10,11 @@ namespace SixLabors.ImageSharp.Formats.Jpeg.Components.Decoder
     /// </summary>
     internal interface IJpegComponent
     {
+        /// <summary>
+        /// Gets the component id.
+        /// </summary>
+        public byte Id { get; }
```

Comment:
The existing interface members don't have the access modifier specified.
It's not strictly needed, but C# allows this (since C# 8 I think).

At least it should be consistent withing this file.